backend/typescript/services/implementations/camperService.ts

```diff
@@ -288,12 +289,14 @@ class CamperService implements ICamperService {
           lastName: camper.lastName,
           age: camper.age,
           allergies: camper.allergies,
-          earlyDropoff: camper.earlyDropoff.map((date) => date.toString()),
-          latePickup: camper.latePickup.map((date) => date.toString()),
+          earlyDropoff: camper.earlyDropoff.map((date) => date?.toString()),
```

Comment:
Did you make these changes? There seem to be a bunch of changes that are not related to your ticket?